Moblin/Various/Model.swift

```diff
@@ -1850,6 +1851,7 @@ final class Model: ObservableObject {
     }
 
     private func updateChat() {
+        pausedChatPostsCount = pausedChatPosts.count
```

Comment:
Move it into the if-statement below, so it is only updated when chat is paused.

Reply:
This might introduce bugs when the chat is paused the second time as the counter is not reset to zero. So you would have to reset this to 0 with another assignment.

Reply:
so for the first 200ms in second pause the count would be incorrect if I use your method

Reply:
hmm true, let's keep it like this now and we can improve it later